WebNetSample/Views/Categories/Index.cshtml

```diff
@@ -2,12 +2,56 @@
 @model IEnumerable<Category>
 
 @{
-    ViewData["Title"] = "Categories Page";
+    ViewData["Title"] = "Detailed Category Page";
 }
 
-<ul>
-    @foreach (var item in Model)
-    {
-        <li>@item.Name</li>
-    }
-</ul>
\ No newline at end of file
+<p>
+    <a class="btn btn-success" asp-controller="Categories" asp-action="Add">NEW CATEGORY</a>
+</p>
+
+<table class="table table-striped table-hover">
+
+    <thead>
+        <tr>
+            <th colspan="7" class="text-center bg-dark text-light">CATEGORIES WITH DETAILS</th>
```

Comment:
CSS trick: you can easily use [`text-transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) CSS property to make you text format in every way you need, just keep in mind that ;)